frontend/server/src/Controllers/Run.php

```diff
@@ -1050,7 +1198,8 @@ public static function apiDownload(\OmegaUp\Request $r): void {
     public static function downloadSubmission(
         string $guid,
         \OmegaUp\DAO\VO\Identities $identity,
-        bool $passthru
+        bool $passthru,
+        bool $showDiff = false
```

Comment:
aquí a este método no le importa si se está mostrando el diff o no. más bien lo que se desea controlar es si se va a ignorar la prueba de autenticación. entonces qué te parece mejor hacer

```suggestion
        bool $skipAuthorization = false
```

?

Reply:
Listo